app/views/posts/index.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1>Posts</h1>
-<p>This is to display all posts</p>
+<% #This page displays all the post of a user %>
+
+<%= image_tag @user.photo, alt: 'user photo', class: 'users-posts' %>
+<h1><%= @user.name %></h1>
+Number of posts: <%= @user.posts_counter %>
+
+<ul>
+  <% @user.posts.each_with_index do |post, index| %>
+    <li>
+      Post <%= index + 1 %> <br><br>
+      <%= link_to post.title, user_post_path(@user, post) %>
+      <br>
+      Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+      
+      <ul>
+        <% post.recent_comments.each do |comment| %>
+          <li><%= @user.name %>: <%= comment.text %></li>
+        <% end %>
+      </ul>
+    </li>
+  <% end %>
```

Comment:
- [ ] Let's ensure that **See all Post** link is present and opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)